api/src/routes/protected/user.ts

```diff
@@ -682,7 +682,8 @@ export const userGetRoutes: FastifyPluginCallbackTypebox = (
               name: name ?? '',
               theme: theme ?? 'default',
               twitter: normalizeTwitter(twitter),
-              username: usernameDisplay || username,
+              username,
+              usernameDisplay,
```

Comment:
```suggestion
              usernameDisplay: usernameDisplay || username,
```
This simplifies the types and is, I think, the default we want.